### DISCUSSION TITLE
Auto hover first element on searchable select

### DISCUSSION THREAD

**Discussion by Mau-MD** [mantinedev/mantine#2372] [Q&A]
When using the `Select` component with the `searchable` prop enabled, it doesn't hover anything on search. 

<img width="851" alt="image" src="https://user-images.githubusercontent.com/74751751/188704034-2299b2f0-f305-4e6c-93ae-fd92a79017ce.png">

It would be extremely nice to automatically hover the first element on the search list to just press enter instead of pressing the down arrow and then enter. Similar to what `react-select` does

I haven't been able to implement this feature. Is it even possible? Thanks!

**Comment by rtivital** [comment]
Hi @Mau-MD, see #2272
